### ISSUE TITLE
Plone Deployment: what to do after `make run-playbook`?

### ISSUE THREAD

**Issue by stevepiercy** [plone/training#701]
I was able to proceed through [`make run-playbook`](https://training.plone.org/plone-deployment/provision.html#run-playbook), but now I don't know what to do next. I tried visiting both http://localhost:3000/ and http://localhost:8080/ but there was no response.

I think the answer was said in person but not in writing 😢 . @jensens @davisagli @sneridagh @ericof 

I think I am supposed to just continue, setting up the For Production environment, then move on to the Deployment chapter. It would be nice to get confirmation, so I can update the Training docs with these expectations.

ref: https://github.com/collective/cookiecutter-plone-starter/issues/42

**Comment by stevepiercy** [comment]
I plowed forward, completing the steps in [Deploy](https://training.plone.org/plone-deployment/deploy.html).

When I visited https://plone-conference.localhost, I got a 404 error response, `404 page not found`.

I checked all the log files, and only `make logs-webserver` showed anything.

```console
==> Logs for webserver in dev 
plone_traefik.1.r030pvd7jhyz@vagrant.localhost    | time="2022-12-26T23:14:31Z" level=info msg="Configuration loaded from flags."
plone_traefik.1.r030pvd7jhyz@vagrant.localhost    | time="2022-12-26T23:14:31Z" level=error msg="Cannot start the provider *file.Provider: error reading configuration file: /config/traefik_config.yaml - open /config/traefik_config.yaml: no such file or directory"
plone_traefik.1.r030pvd7jhyz@vagrant.localhost    | 10.0.0.2 - - [26/Dec/2022:23:14:48 +0000] "GET / HTTP/1.1" - - "-" "-" 1 "-" "-" 0ms
plone_traefik.1.r030pvd7jhyz@vagrant.localhost    | 10.0.0.2 - - [26/Dec/2022:23:14:49 +0000] "GET /favicon.ico HTTP/1.1" 301 17 "-" "-" 2 "generic-https-redirect@docker" "-" 0ms
plone_traefik.1.r030pvd7jhyz@vagrant.localhost    | 10.0.0.2 - - [26/Dec/2022:23:15:10 +0000] "GET / HTTP/1.1" 301 17 "-" "-" 3 "generic-https-redirect@docker" "-" 0ms
plone_traefik.1.r030pvd7jhyz@vagrant.localhost    | 10.0.0.2 - - [26/Dec/2022:23:15:17 +0000] "GET / HTTP/2.0" 404 19 "-" "-" 4 "-" "-" 0ms
```

When I visit https://traefik.localhost/, and login with username and password of `admin`, I am redirected to the dashboard.

I can ssh into the local Plone server, but I don't know what to look for.

```shell
ssh plone@127.0.0.1 -p 2222
```

In `devops/stacks/plone.yml`, I see a couple of lines of interest. How are these two images supposed to be created?

```yaml
  frontend:
    image: ghcr.io/stevepiercy/plone-conference-frontend:latest
# snip
  backend:
    image: ghcr.io/stevepiercy/plone-conference-backend:latest
```

I am not sure what to check next. Any ideas?

**Comment by ericof** [comment]
@stevepiercy After `make run-playbook` you run `make deploy` (If you already have the images built on the CI) and then point your browser to `https://ploneconference.localhost`

**Comment by stevepiercy** [comment]
@ericof I did that per the training.

The training does not provide instructions for how to build an image on GitHub. This is the only mention of having images on CI, and itself is not clear:

> GitHub and GitHub actions
>
> Plone's development happens on GitHub, and the community is increasingly adopting GitHub actions in their new packages Concepts explained here could be adapted to other environments like GitLab and Jenkins

I found this Quick Start Guide for GitHub Packages: https://docs.github.com/en/packages/quickstart

Is that the correct guide to follow for this training?

I might have zoned out while fighting my old laptop at PloneConf, and missed when this was discussed during the training.
